introspective_ORB_SLAM/src/torch_helpers.cpp

```diff
@@ -19,76 +19,89 @@
 
 #include "torch_helpers.h"
 
-
 namespace ORB_SLAM2 {
 
-std::string GetImageType(const cv::Mat& img, bool more_info) {
-    std::string r;
```

Comment:
Good practice to only declare things right before you use them, so this is moved to right before the switch